src/Components/RecoProductList/RecoProduct/RecoProduct.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import 'Components/RecoProductList/RecoProduct/RecoProduct.scss';
+import React from 'react';
```

Comment:
```suggestion
import React,{ useState } from 'react';
import './RecoProduct.scss';
```
import 순서 컨벤션에 맞게 수정해주세요! 
사소한 부분이라 놓치기 쉽지만 이런 컨벤션 준수들이 모여서 좀 더 가독성 좋은 코드가 됩니다. 
React, {useState}는 같은 곳에서 import 해주기 때문에 같이 `,` 로 구분해서 쓸 수 있습니다!
세션 자료 및 아래 순서 참고해주세요.
- 라이브러리
    - React 관련 패키지
    - 외부 라이브러리
- 컴포넌트
    - 공통 컴포넌트 → 먼 컴포넌트 → 가까운 컴포넌트
- 함수, 변수 및 설정 파일
- 사진 등 미디어 파일(`.png`)
- css 파일 (.`scss`)

Reply:
네 수정했습니다! 감사합니다!